File: Palindrome_Tester.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        Console.Write("Enter a word: ");
        string word = Console.ReadLine();
        string reversedWord = Reverse(word);

        if (word == reversedWord)
        {
            Console.WriteLine("The word is a palindrome.");
        }
        else
        {
            Console.WriteLine("The word is not a palindrome.");
        }
    }

    static string Reverse(string s)
    {
        char[] charArray = s.ToCharArray();
        Array.Reverse(charArray);
        return new string(charArray);
    }
}

File: Palindrome_Tester.py
def is_palindrome(input_string):
    return input_string == input_string[::-1]

while True:
    user_input = input("Enter a word to check if it's a palindrome or 'q' to quit: ")
    if user_input.lower() == 'q':
        break
    else:
        if is_palindrome(user_input):
            print("It's a palindrome!")
        else:
            print("It's not a palindrome.")